requirements.txt

```diff
@@ -4,7 +4,7 @@ fastapi == 0.68.0
 # pydantic == 1.7.4
 gunicorn == 20.0.4
 rcsb.utils.io >= 1.17
-rcsb.utils.chem >= 0.82,<1.0
+rcsb.utils.chem >= 0.83,<1.0
```

Comment:
@dmyersturnbull Why the `<1.0`?

Reply:
@piehld The two projects very tightly coupled, and we could expect a hypothetical `1.0` to be backwards-incompatible. Granted, it's not actually using semver, and both projects should be abandoned soonish (so this makes little difference).